web-api/src/utils.rs

```diff
@@ -53,3 +53,22 @@ pub(crate) fn serialize_to_ndjson(
     }
     Ok(body)
 }
+
+pub(crate) mod serde_duration_as_secs {
```

Comment:
:wrench:  Why rename this just to save 4 characters in name which anyway gets auto-completed.

Reply:
Is shorter, faster to read, and uniform with `as_secs`/`from_secs` from `Duration`.

Reply:
`as_secs`/`from_secs` are like `FromStr::Err` named in a way they likely wouldn't be named anymore, just around rust ~1.3(?) days there was much less clearity about naming schema.

And just because other places use bad style doesn't mean we have to do so, too.

Reply:
E.g. chrono does use seconds, nanoseconds, etc. (except in one helper trait for rounding time :shrug: )

Reply:
I'll change it I don't care about it too much.
I don't think is a bad convention, it is clear and completely unambiguous to everyone, with no drawbacks. Since it is used in a literal string with serde is not touched by rust-analyzer and we have to write more for no reason.

Reply:
rust analizer works for string literals in macros which are turned in idents, or at least in VSCode it does

Reply:
rust-analyzer 1.69.0 (84c898d 2023-04-16) is not able to rename or autocomplete in those. Do you use any particular option?